src/compas_fab/robots/semantics.py

```diff
@@ -128,6 +138,17 @@ def get_base_link_name(self, group=None):
             group = self.main_group_name
         return self._group_dict[group]["links"][0]
 
+    def get_all_configurable_joints(self):
```

Comment:
How is this method different from `model.get_configurable_joints()`?

Reply:
it would exclude the passive ones. this information is only stored in the semantics

Reply:
👍  but couldn't we reuse the model method and then filter, instead of this longer method?

Reply:
sure!